src/IntercomContacts.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Intercom;
+
+use Http\Client\Exception;
+use stdClass;
+
+class IntercomContacts extends IntercomResource
+{
+    /**
+     * Returns list of Contacts that match search query.
+     *
+     * @see     https://developers.intercom.com/reference#search-for-contacts
+     * @param   array $options
+     * @return  stdClass
+     * @throws  Exception
+     */
+    public function searchContacts(array $options)
```

Comment:
Considering `Contacts` is already the name of this class, I'd rename this to `search`